README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
@kevinsawicki I don't understand in this case why do you have https://github.com/atom/atom/blob/f48069c5f848b5ae624b110a1df7800e919eb142/src/browser/auto-updater-win32.coffee ?